CMakePresets.json

```diff
@@ -88,6 +88,11 @@
       "configuration": "Release",
       "targets": ["docs", "doctest", "cleanup_docs_html"]
     },
+    {
+      "name": "package-docs",
+      "inherits": "docs",
+      "targets": ["docs", "doctest", "linkcheck" ,"cleanup_docs_html"]
+    },
```

Comment:
I presume you split this up because `linkcheck` is taking too long for a normal run?

Reply:
Yes. It doesn't take super long but I don't think it is worth slowing down CI even more for this.